src/plots/layout_attributes.js

```diff
@@ -140,6 +140,36 @@ module.exports = {
         }),
         editType: 'layoutstyle'
     },
+    uniformtext: {
+        mode: {
+            valType: 'enumerated',
+            values: [false, 'hide', 'show'],
+            dflt: false,
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Determines how the font size for various text',
+                'elements are uniformed between each trace type.',
+                'If the computed text sizes were smaller than',
+                'the minimum size defined by `minsize`',
+                'using *hide* option hides the text; and',
+                'using *show* option shows the text without further downscaling.',
+                'Please note that if the size defined by `minsize` is greater than',
+                'the font size defined by trace, the `minsize` would be used.'
+            ].join(' ')
+        },
+        minsize: {
+            valType: 'number',
+            min: 0,
```

Comment:
Hmm. I thought we agreed on allowing `minsize: 0` and making `minsize`  have `min: -1`

Reply:
Disabling `uniformtext` using `mode: false` (or possibly `none` ?) made the process much simpler.
In addition we may want to add other options for unifying e.g. rotations in future; and in that sense depending on the value of `minsize` is not great.

Reply:
what would `-1` mean here?

Reply:
Right now. Nothing. Any value less than zero is not allowed as treated as default zero.

Reply:
Ok nevermind - I think I got confused by your previous iteration where you have `min: 1`.

Closing.